waku/v2/protocol/waku_store/protocol.nim

```diff
@@ -133,35 +130,34 @@ proc findMessages(w: WakuStore, query: HistoryQuery): HistoryResponse {.gcsafe.}
 proc init*(ws: WakuStore, capacity = StoreDefaultCapacity) =
 
   proc handler(conn: Connection, proto: string) {.async.} =
-    var message = await conn.readLp(MaxRpcSize.int)
-    var res = HistoryRPC.init(message)
-    if res.isErr:
-      error "failed to decode rpc"
+    let buf = await conn.readLp(MaxRpcSize.int)
```

Comment:
Non-blocking: do you think changes in naming like this (`message` > `buf`) should be done in other files too?

<img width="423" alt="image" src="https://user-images.githubusercontent.com/13265126/188023223-eb02b4fc-c5f8-4961-a286-acfbe70c7259.png">

Reply:
Yes, i fully agree, we'll do it as we do small refactoring on the different protocol implementations 😁